### PR TITLE
Fixes #30254, #30260 - Add DSL docs for errata/CVE

### PR DESCRIPTION
**Pull request by ofedoren**
Needs https://github.com/Katello/katello/pull/8801

### CODE REVIEW

File: app/models/katello/erratum_cve.rb [hunk 1]
```diff
@@ -2,6 +2,14 @@ module Katello
   class ErratumCve < Katello::Model
     belongs_to :erratum, :inverse_of => :cves, :class_name => 'Katello::Erratum'
 
+    apipie :class, desc: 'A class representing Erratum CVE object' do
+      name 'Erratum CVE'
+      refs 'ErratumCve'
+      sections only: %w[all additional]
+      property :erratum, 'Erratum', desc: 'Returns Erratum object associated with this CVE'
```
Comment (jlsherrill) [inline app/models/katello/erratum_cve.rb:9]: I got Uninitialized Constnat 'Erratum' here, this needs "Katello::" on the front of it

File: app/models/katello/erratum.rb [hunk 2]
```diff
@@ -161,6 +161,19 @@ def self.find_by_modular(_key, operator, value)
...
+      name 'Erratum'
+      refs 'Erratum'
+      sections only: %w[all additional]
+      property :errata_id, String, desc: 'Returns errata identifier, e.g. "RHSA-1999-1231"'
+      property :errata_type, String, desc: 'Returns errata type, e.g. "security"'
+      property :issued, Date, desc: 'Returns the date of issue for the errata'
+      property :created_at, ActiveSupport::TimeWithZone, desc: 'Returns the time when the errata was created'
+      property :severity, String, desc: 'Returns severity of the errata, e.g. "Critical"'
+      property :package_names, array_of: String, desc: 'Returns names of packages the errata can be applied to'
+      property :cves, array_of: 'ErratumCve', desc: 'Returns CVEs associated with the errata'
```
Comment (jlsherrill) [inline app/models/katello/erratum.rb:174]: Same thing here, Katello:: needs to be on the front

**Comment by jlsherrill** [comment]
Would it be possible to add a test which runs the 'apipie_dsl:cache' rake task?

**Comment by ofedoren** [comment]
@jlsherrill, forgot to mention that if https://github.com/Katello/katello/pull/8815 is merged, then it should be ok.

**Comment by jlsherrill** [comment]
@ofedoren isn't this #8815 ?

**Comment by ofedoren** [comment]
> @ofedoren isn't this #8815 ?

Dummy me, sure.. Let me check what's going on.

I didn't want to put 'Katello::' scope since it might be confusing for Satellite users. Actually it shouldn't  look for constants untill all the constants are loaded by Rails. Let me re-check...

**Comment by jlsherrill** [comment]
Here's the traceback during the rake task run:

```
ActionView::Template::Error: uninitialized constant ErratumCve
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/active_support.rb:80:in `block in load_missing_constant'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/active_support.rb:9:in `without_bootsnap_cache'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/active_support.rb:80:in `rescue in load_missing_constant'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/active_support.rb:59:in `load_missing_constant'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/inflector/methods.rb:282:in `const_get'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/inflector/methods.rb:282:in `block in constantize'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/inflector/methods.rb:280:in `each'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/inflector/methods.rb:280:in `inject'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/inflector/methods.rb:280:in `constantize'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/core_ext/string/inflections.rb:68:in `constantize'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/app/helpers/apipie_dsl_helper.rb:84:in `class_references'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/app/views/apipie_dsl/apipie_dsls/_returns.html.erb:50:in `_fdcd37a8724bb2a48e11fd2b978dffb0'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:274:in `_run'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:185:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:182:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:386:in `instrument_render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:183:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/deface-1.5.3/lib/deface/action_view_extensions.rb:43:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:358:in `block in render_partial'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:89:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:88:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:347:in `render_partial'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:317:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:65:in `render_partial_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:27:in `render_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:22:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:34:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:304:in `in_rendering_context'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:30:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/app/views/apipie_dsl/apipie_dsls/_property_detail.html.erb:33:in `_1c8da4429ae8d11de7ab9da5cc1d6e0c'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:274:in `_run'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:185:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:182:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:386:in `instrument_render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:183:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/deface-1.5.3/lib/deface/action_view_extensions.rb:43:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:358:in `block in render_partial'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:89:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:88:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:347:in `render_partial'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:317:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:65:in `render_partial_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:27:in `render_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:22:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:34:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:304:in `in_rendering_context'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:30:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/app/views/apipie_dsl/apipie_dsls/_property.html.erb:19:in `_3d84acad9bdc180ca8a986c99a427e15'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:274:in `_run'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:185:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:182:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:386:in `instrument_render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:183:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/deface-1.5.3/lib/deface/action_view_extensions.rb:43:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:452:in `block in collection_with_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:448:in `map'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:448:in `collection_with_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activerecord-6.0.2.2/lib/active_record/railties/collection_cache_association_loading.rb:30:in `collection_with_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:337:in `block (2 levels) in render_collection'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer/collection_caching.rb:15:in `cache_collection_render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:336:in `block in render_collection'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:89:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:88:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:325:in `render_collection'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/partial_renderer.rb:315:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:65:in `render_partial_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:27:in `render_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:22:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:34:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:304:in `in_rendering_context'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:30:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/app/views/apipie_dsl/apipie_dsls/class.html.erb:55:in `_64527002051d3b5f8abdcc864aafdc1d'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:274:in `_run'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:185:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:182:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:386:in `instrument_render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/template.rb:183:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/deface-1.5.3/lib/deface/action_view_extensions.rb:43:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/template_renderer.rb:59:in `block (2 levels) in render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:89:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `block in instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/activesupport-6.0.2.2/lib/active_support/notifications.rb:180:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/abstract_renderer.rb:88:in `instrument'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/template_renderer.rb:58:in `block in render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/template_renderer.rb:66:in `render_with_layout'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/template_renderer.rb:57:in `render_template'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/template_renderer.rb:13:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:61:in `render_template_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:29:in `render_to_object'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/renderer/renderer.rb:22:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:34:in `block in render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/base.rb:304:in `in_rendering_context'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/actionview-6.0.2.2/lib/action_view/helpers/rendering_helper.rb:30:in `render'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:38:in `block in render_page'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:34:in `open'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:34:in `render_page'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:78:in `block in generate_class_pages'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:70:in `each'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:70:in `generate_class_pages'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:30:in `block (6 levels) in <top (required)>'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:23:in `each'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:23:in `block (5 levels) in <top (required)>'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:21:in `each'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:21:in `block (4 levels) in <top (required)>'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:15:in `each'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:15:in `block (3 levels) in <top (required)>'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks_utils.rb:126:in `with_loaded_documentation'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/apipie-dsl-2.2.7/lib/apipie_dsl/tasks/apipie_dsl/cache.rake:13:in `block (2 levels) in <top (required)>'
/home/vagrant/git/foreman/.vendor/ruby/2.5.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `load'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `kernel_load'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:28:in `run'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:424:in `exec'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:27:in `dispatch'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/cli.rb:18:in `start'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/exe/bundle:30:in `block in <top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/bundler-1.16.1/exe/bundle:22:in `<top (required)>'
/opt/rh/rh-ruby25/root/usr/bin/bundle:23:in `load'
/opt/rh/rh-ruby25/root/usr/bin/bundle:23:in `<main>'

```

**Comment by ofedoren** [comment]
@jlsherrill, I found why it happens: this PR needs rebase since the required PR was merged. To work properly this needs https://github.com/ofedoren/katello/blob/e7205fe19b9ae4a45a6c4f65196a349c6ee148b8/lib/katello/plugin.rb#L230-L232. I'll rebase PRs.

**Comment by jlsherrill** [comment]
@ofedoren i had rebased locally when testing the pr

**Comment by ofedoren** [comment]
@jlsherrill and now it's weird.. I've re-checked again (foreman/katello both latest develop + rebased) with 3 ways of getting the docs and encountered no errors:

ScreenShot-1594995066900

**Comment by jlsherrill** [comment]
[test katello]

**Pull request merged**